content/introduction/_index.md

```diff
@@ -49,57 +53,61 @@ As mentioned, SODA Architecture is getting refined and optimized for different a
+  More details [here](https://sodafoundation.io/projects/delfin-soda-infrastructure-manager/)
 
 - **SODA Plugin**
-SODA North-Bound Plugin Project focuses to extend all the industry platforms and application solutions to interface with SODA API or be compliant with it.
-More details [here](https://sodafoundation.io/projects/soda-plugins/)
+  SODA North-Bound Plugin Project focuses to extend all the industry platforms and application solutions to interface with SODA API or be compliant with it.
+  More details [here](https://sodafoundation.io/projects/soda-plugins/)
 
 - **SODA Multicloud**
-SODA Multicloud project provides a cloud vendor agnostic data management for hybrid cloud, crosscloud or incloud. It can be hosted on prem or cloud native.
-More details [here](https://sodafoundation.io/projects/soda-multicloud-dds/)
+  SODA Multicloud project provides a cloud vendor agnostic data management for hybrid cloud, crosscloud or incloud. It can be hosted on prem or cloud native.
+  More details [here](https://sodafoundation.io/projects/soda-multicloud-dds/)
 
 - **SODA Orchestration**
-The Orchestration framework provides flexibility to use existing workflows or define customized workflows to get the simplified execution of tasks.
-More details [here](https://sodafoundation.io/projects/soda-experiments/)
-
+  The Orchestration framework provides flexibility to use existing workflows or define customized workflows to get the simplified execution of tasks.
+  More details [here](https://sodafoundation.io/projects/soda-experiments/)
 
 ## Official Charter
+
 The official charter for SODA Foundation under Linux Foundation can be found [here](https://sodafoundation.io/the-foundation/charter/)
 
 **Want to be part of this initiative?**
 Please join us to collaborate, research, develop and lead for unified data framework and standards.
 Contact us through any of the following channels:
 
- - [SODA Website](https://sodafoundation.io/the-foundation/join/)  
- - [SODA Slack](https://sodafoundation.io/slack)
- - [SODA Twitter](https://twitter.com/sodafoundation)  
- - [SODA Mailing List](https://lists.sodafoundation.io)
- - [SODA LinkedIn](https://www.linkedin.com/company/sodafoundation/)
+<ul class="social-list">
+  <li class="social-list-item"><a href="https://sodafoundation.io/the-foundation/join/"><i class="fas fa-globe"></i> SODA Website</a></li>
+  <li class="social-list-item"><a href="https://sodafoundation.io/slack"><i class="fab fa-slack"></i> SODA Slack</a></li>
+  <li class="social-list-item"><a href="https://twitter.com/sodafoundation"><i class="fab fa-twitter"></i> SODA Twitter</a></li>
+  <li class="social-list-item"><a href="https://lists.sodafoundation.io"><i class="fas fa-envelope"></i> SODA Mailing List</a></li>
+  <li class="social-list-item"><a href="https://www.linkedin.com/company/sodafoundation/"><i class="fab fa-linkedin"></i> SODA LinkedIn</a></li>
+</ul>
+
```

Comment:
We can also add a new link:
To know the introduction and how to contribute, you can refer [SODA Starter](https://bit.ly/soda-starter)